frontend/packages/console-app/console-extensions.json

```diff
@@ -21,6 +21,13 @@
       "importRedirectURL": { "$codeRef": "perspective.getImportRedirectURL" }
     }
   },
+  {
+    "type": "SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.guided-tour",
```

Comment:
I appreciate the humor but I'd probably just call this something like `INTERNAL_DO_NOT_USE`